src/ipc/server.rs

```diff
@@ -446,6 +446,86 @@ async fn process(ctx: &ClientCtx, request: Request) -> Reply {
             let is_open = state.overview.is_open;
             Response::OverviewState(Overview { is_open })
         }
+        Request::Binds => {
+            let (tx, rx) = async_channel::bounded(1);
+            ctx.event_loop.insert_idle(move |state| {
+                let binds = state
+                    .niri
+                    .config
+                    .borrow()
+                    .binds
+                    .0
+                    .iter()
+                    .map(|b| {
+                        let trigger = match b.key.trigger {
+                            niri_config::Trigger::Keysym(k) => k.name().map(String::from),
```

Comment:
There should be a better way to format the trigger key here, instead of returning an `Option`.